test_runner/regress/test_lsn_mapping.py

```diff
@@ -108,6 +109,43 @@ def test_lsn_mapping(neon_env_builder: NeonEnvBuilder):
         assert Lsn(result["lsn"]) >= last_flush_lsn
 
 
+#
+# Test if cancelled pageserver get_lsn_by_timestamp request is correctly handled.
+#
+# Added as an effort to improve error handling and avoid full anyhow backtrace
+# https://github.com/neondatabase/neon/pull/7949
+#
+def test_get_lsn_by_timestamp_cancelled(neon_env_builder: NeonEnvBuilder):
+    env = neon_env_builder.init_start()
```

Comment:
```suggestion
def test_get_lsn_by_timestamp_cancelled(neon_env_builder: NeonEnvBuilder):
    """
    Test if cancelled pageserver get_lsn_by_timestamp request is correctly handled.
    Added as an effort to improve error handling and avoid full anyhow backtrace
    """
    env = neon_env_builder.init_start()
```

IMHO no need for:
- surrounding `#`
- better as a python whatever this is called, because it will be picked up for the allure test report
- linking the PR, it will be apparent when using squash and merge in the subject of the commit